src/FormField/RegisteredMFAMethodListField.php

```diff
@@ -44,21 +49,21 @@ public function getSchemaDataDefaults()
         $generator = SchemaGenerator::create();
 
         if (!$this->value && $this->getForm() && $this->getForm()->getRecord() instanceof Member) {
-            $member = $this->getForm()->getRecord();
+            $this->member = $this->getForm()->getRecord();
         } else {
-            $member = DataObject::get_by_id(Member::class, $this->value);
+            $this->member = DataObject::get_by_id(Member::class, $this->value);
         }
```

Comment:
Looking at this again, I think it would be better to convert this part into a `private` method that returns the member record, and can be called from both `getSchemaDataDefaults()` and `getBackupMethod()`.

That way we never have to fall back on `?? Security::getCurrentUser()` if `getBackupMethod()` is called first.